pkg/network/protocols/http/ebpf_javatls.go

```diff
@@ -78,7 +103,37 @@ func (p *JavaTLSProgram) GetAllUndefinedProbes() (probeList []manager.ProbeIdent
 	return
 }
 
+func allowBlockAttachment(pid uint32) (reject bool) {
```

Comment:
the name of the function is misleading, change so it will be clearer

Reply:
done